data/event-schemas/schema/core-event-schema/sdp_stream_metadata.yaml

```diff
@@ -0,0 +1,27 @@
+type: object
```

Comment:
I don't think this belongs in `core-event-schema`

Reply:
I agree. Since it doesn't seem to belong directly in `schemas` either, because it only contains full event schemas, I took the liberty to create a `components` directory (named like this for similarity with the property that includes reusable bits in the OpenAPI spec)